mslib/utils/config.py

```diff
@@ -500,9 +498,7 @@ def load_settings_qsettings(tag, default_settings=None, ignore_test=False):
         return default_settings
 
     settings = {}
-
-    q_settings = QtCore.QSettings(os.path.join(constants.MSUI_CONFIG_PATH, "msui-core.conf"),
-                                  QtCore.QSettings.IniFormat)
+    q_settings = QtCore.QSettings(constants.MSUI_CORE_FILE_PATH, QtCore.QSettings.IniFormat)
```

Comment:
https://github.com/Open-MSS/MSS/issues/2148

There was a mismatch for fs by os path joining. Because we check if pytest is in sys.modules I assume the finding I had were by setting the env var myself.